README.md

```diff
@@ -67,7 +67,7 @@ brew install docker-compose
 
 ##### Setup
 ```
-docker-compose up
+docker-compose -f .docker/docker-compose.yml up
```

Comment:
Can we add a couple of helpers to the `bin` directory for this?

`bin/docker-up` and `bin/docker-down` maybe?

Reply:
@harimohanraj89 how about adding a make file 

```
docker-up:
	docker-compose -f .docker/docker-compose.yml up
docker-down:
	docker-compose -f  .docker/docker-compose.yml down --volumes
```

Reply:
We've got `bin` scripts for other things right now, I'd rather move everything wholesale later, rather than have some stuff live there and some stuff live in a makefile, what do you think?

Reply:
sure